ports/esp32/machine_sdcard.c

```diff
@@ -278,6 +290,15 @@ STATIC mp_obj_t sd_deinit(mp_obj_t self_in) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(sd_deinit_obj, sd_deinit);
 
+STATIC mp_obj_t sd_card_print_info(mp_obj_t self_in) {
+    sdcard_card_obj_t *self = self_in;
+
+    sdmmc_card_print_info(stdout, &(self->card));
+
+    return mp_const_none;
+}
+STATIC MP_DEFINE_CONST_FUN_OBJ_1(sd_card_print_info_obj, sd_card_print_info);
```

Comment:
This is an unrelated addition and shouldn't be in this PR (it requires a separate discussion).

Reply:
OK!